clients/p4rt-ctl/p4rt-ctl.in

```diff
@@ -769,6 +791,58 @@ def p4ctl_show(client, bridge):
 def p4ctl_set_pipe(client, bridge, device_config, p4info):
     client.set_fwd_pipe_config(p4info, device_config)
 
+class TapDevice:
+    def __init__(self, tap_name):
+
+        self.tap_name = tap_name
+
+        TUNSETIFF = 0x400454CA
```

Comment:
Can we use a macro to define these contstants instead and add a comment that it's from linux/if_tun.h? https://peps.python.org/pep-0591/

Reply:
same comment applicable for all the values below

Reply:
As I understand, uppercase variable names are used to indicate that a variable's value should be treated as constant. I understand the variable is not truly immutable but that's the convention used to declare consts.

Thanks for the reference on using 'Final'. I now, learnt that there is a way to enforce immutability.  However, it involves importing an additional module which might be a overkill in this specific case.